servicetalk-log4j2-mdc/src/main/java/io/servicetalk/log4j2/mdc/DefaultServiceTalkThreadContextMap.java

```diff
@@ -17,8 +17,41 @@
 
 import io.servicetalk.log4j2.mdc.utils.ServiceTalkThreadContextMap;
 
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
 /**
- * Empty subclass to differentiate uses of MDC.
+ * Subclass which uses the {@link io.servicetalk.concurrent.api.CapturedContext} mechanism for propagation.
  */
 public class DefaultServiceTalkThreadContextMap extends ServiceTalkThreadContextMap {
+
+    private static final String ENABLE_PROPERTY_NAME = "io.servicetalk.log4j2.mdc.capturedContextStorage";
```

Comment:
Consider adding `FIXME: 0.43` to remove this property and maybe even make this class `final`